libimage/events.go

```diff
@@ -2,13 +2,13 @@ package libimage
 
 import "time"
 
-// EventType indicates the type of an event.  Currrently, there is only one
+// EventType indicates the type of an event.  Currently, there is only one
 // supported type for container image but we may add more (e.g., for manifest
 // lists) in the future.
 type EventType int
 
 const (
-	// EventTypeUnknow is an unitialized EventType.
+	// EventTypeUnknow is an uninitialized EventType.
```

Comment:
Missing the "n" at the end of "EventTypeUnknown".

Reply:
Surprised that codespell did not catch that one.